cli-toolchain/graphql.md

```diff
@@ -2379,6 +2379,157 @@ Here is a complete list of searchable operations per GraphQL type supported as o
 | Boolean | `eq`, `ne`      |
 
 
+
+### @predictions
+
+The `@predictions` directive allows one to query an orchestration of AI/ML services such as Amazon Rekognition, Amazon Translate, and/or Amazon Polly.
+
+> Note: Support for adding the `@predictions` directive uses the s3 storage bucket which is configured via the CLI. At the moment this directive works only with objects located within `public/`.
+
+#### Definition
+The supported actions in this directive are included in the definition.
+
+```
+# where the parent this field is defined on is a query type
+        directive @predictions(actions: [PredictionsActions!]!) on FIELD_DEFINITION
+        enum PredictionsActions {
+          identifyText
+          identifyLabels
+          convertTextToSpeech
+          translateText
+        }
+```
+
+#### Usage
+
+
+Given the following schema a query operation is defined which will do the following with the provided image.
+
+- Identify text from the image
+- Translate the text from that image
+- Synthesize speech from the translated text.
+
+```graphql
+type Query {
+  speakTranslatedImageText: String @predictions(actions: [
+    identifyText
+    translateText
+    convertTextToSpeech
+  ])
+}
+```
+
+An example of that query will look like:
+
+```graphql
+query SpeakTranslatedImageText($input: SpeakTranslatedImageTextInput!) {
+  speakTranslatedImageText(input: {
+    identifyText: {
+      key: "myimage.jpg"
+    }
+    translateText: {
+      sourceLanguage: "en"
+      targetLanguage: "es"
+    }
+    convertTextToSpeech: {
+      voiceID: "Conchita"
+    }
+  })
+}
+```
+
+A code example of this using the JS Library:
```

Comment:
I noticed in the regular Predictions category docs you have to [configure a pluggable](https://aws-amplify.github.io/docs/js/predictions#configure-your-app), is this not necessary when using this new API?

Reply:
Nope this is not necessary - the resources created are used so AppSync has permissions to make those calls on behalf of the caller.